assets/ts/stop/__tests__/DepartureCardTest.tsx

```diff
@@ -149,10 +204,8 @@ describe("DepartureCard", () => {
     );
 
     await waitFor(() => {
-      const suspensionBadges = screen.getAllByText("No Service");
-      expect(suspensionBadges.length).toBe(Object.keys(routePatterns).length);
-      expect(screen.queryByText("Shuttle Service")).toBeNull();
-      expect(screen.queryByText("Detour")).toBeNull();
```

Comment:
These are no longer applicable as they only occur when there are no departures. Because we have a departure, the matching alert is 'Detour'.

Reply:
I don't think so, I think that because everything was route pattern based previously, they just didn't pass in any departures in the tests. But, now that everything is departure based, you have to pass in a departure or it falls back to the default route pattern behavior of listing it as no departures.

Reply:
Have checked on it and confirmed what I wanted to see 👍🏼